src/components/SubHeader/SubHeader.tsx

```diff
@@ -0,0 +1,22 @@
+import './SubHeader.sass';
+import logo from '@/assets/logo.svg';
+import AuthBtn from '@/components/btns/AuthBtn/AuthBtn';
+import CartBtn from '@components/btns/CartBtn/CartBtn';
+
+const SubHeader: React.FC = () => {
+  return (
+    <div className="subHeader">
+      <div className="subHeader__authPanel">
+        <a href="#">
```

Comment:
Did you try to use ReactRouter links instead of `a` here?

Reply:
The same in `NavMenu`.

Reply:
It was just temporary. But I will add router link now, ok, thanks.

Reply:
Try to write it in a right way from the start.

Reply:
but now it is a problem, because I want to use href="tel:+420336775664"... and for this I need a simple <a> tag, not Link. The same for email.

Reply:
For this cases you can use it.
Of course we have cases where we need pure `a` tag.